README.rst

```diff
@@ -23,6 +23,8 @@ supported Python versions.
 +==========================+==========================+=================================+
 | ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.8``, ``<=3.10``           |
 +--------------------------+--------------------------+---------------------------------+
+| ``2.0.0``                | ``0.15.0``               | ``>=3.8``, ``<=3.10``           |
```

Comment:
```suggestion
| ``2.0.0``                | ``0.15.0``               | ``>=3.8``, ``<=3.11``           |
```

Are we not supporting python 3.11? As per blog seems yes
https://pytorch.org/blog/deprecation-cuda-python-support/